front_end/src/public/js/bootstrap.js

```diff
@@ -0,0 +1,2363 @@
+/*!
+ * Bootstrap v3.3.5 (http://getbootstrap.com)
```

Comment:
Is there a better way to add bootstrap than to check in their code to our repo? submodule? npm install? I don't know, so just asking.